pom.xml

```diff
@@ -65,16 +70,28 @@
     </license>
   </licenses>
 
+  <build>
+    <plugins>
+      <plugin>
+        <groupId>org.codehaus.mojo</groupId>
+        <artifactId>findbugs-maven-plugin</artifactId>
+        <configuration>
+          <excludeFilterFile>findbugs-exclude.xml</excludeFilterFile>
+        </configuration>
+      </plugin>
+    </plugins>
+  </build>
+
   <repositories>
```

Comment:
Nit: For future reference, the parent POM provides automatic detection: https://github.com/jenkinsci/plugin-pom/blob/master/pom.xml#L855-878 (if the file is in the provided path)